tests/threads/testvirtualasynceventwiththreads.nim

```diff
@@ -0,0 +1,27 @@
+discard """
```

Comment:
Maybe it would make sense to put all these tests into a ``virtualevent`` subdirectory?

Reply:
That would be cool, does testament support this?